### PR TITLE
Add italian language for README.md

### PR DESCRIPTION
**Pull request by alessiobenvenuti**

### CODE REVIEW

File: README-it.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Impara a programmare
+> Tutorial, script, frammenti di codice e risorse per aiutarti a conoscere vari linguaggi e diventare uno sviluppatore migliore
+
+[! [Licenza MIT] (https://img.shields.io/badge/License-MIT-blue.svg)] (https://github.com/MichaelCurrin/learn-to-code/blob/master/LICENSE )
```
Comment (MichaelCurrin) [inline README-it.md:4]:
```suggestion
[![Licenza MIT](https://img.shields.io/badge/License-MIT-blue.svg)](https://github.com/MichaelCurrin/learn-to-code/blob/master/LICENSE )
```

File: README-it.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+Migliora le tue abilità di programmazione in vari linguaggi esaminando il contenuto di questo repository. Documento qui gli argomenti mentre imparo ed esploro, in modo da avere un facile riferimento o una guida pratica per me stesso, o nel caso in cui qualcun altro possa trarne beneficio.
+
+## Contenuti in questo progetto
+
+| Titolo | Descrizione |
+| -- | -- |
+| 🤷 [Introduzione assoluta alla programmazione] (/intro_to_coding.md) | Se non hai un background tecnologico e desideri una presentazione della programmazione, inizia da qui.
```
Comment (MichaelCurrin) [inline README-it.md:12]:
```suggestion
| 🤷 [Introduzione assoluta alla programmazione](/intro_to_coding.md) | Se non hai un background tecnologico e desideri una presentazione della programmazione, inizia da qui.
```

File: README-it.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+| 🗺️ [Tabella di marcia] (/roadmaps.md) | Collegamenti a tabelle di marcia in modo da poter scegliere linguaggi e framework da apprendere in un ordine logico. |
```
Comment (MichaelCurrin) [inline README-it.md:13]:
```suggestion
| 🗺️ [Tabella di marcia](/roadmaps.md) | Collegamenti a tabelle di marcia in modo da poter scegliere linguaggi e framework da apprendere in un ordine logico. |
```

File: README-it.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+| 📚 [Risorse] (/resources.md) | Per l'apprendimento dei concetti della programmazione. Include alcuni collegamenti esterni. |
```
Comment (MichaelCurrin) [inline README-it.md:14]:
```suggestion
| 📚 [Risorse](/resources.md) | Per l'apprendimento dei concetti della programmazione. Include alcuni collegamenti esterni. |
```

File: README-it.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+| ⌨ [Guide specifiche dei linguaggi] (/language_guides.md) | Panoramica delle guide dei linguaggi di programmazione in questo progetto, che tu sia un principiante o desideri immergerti in alcuni concetti avanzati. Numerosi collegamenti a risorse esterne come documentazione e corsi online |
```
Comment (MichaelCurrin) [inline README-it.md:15]:
```suggestion
| ⌨ [Guide specifiche dei linguaggi](/language_guides.md) | Panoramica delle guide dei linguaggi di programmazione in questo progetto, che tu sia un principiante o desideri immergerti in alcuni concetti avanzati. Numerosi collegamenti a risorse esterne come documentazione e corsi online |
```

File: README-it.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+| 🌐 [Consigliati] (/recommended.md) | Collegamenti esterni che mi piacciono e che vorrei condividere con gli altri. |
```
Comment (MichaelCurrin) [inline README-it.md:16]:
```suggestion
| 🌐 [Consigliati](/recommended.md) | Collegamenti esterni che mi piacciono e che vorrei condividere con gli altri. |
```

File: README-it.md [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+
+
+## Chi sono?
+
+Maggiori informazioni sulla mia esperienza e sulla mia storia sul mio blog [michaelcurrin.github.io/coding-blog lasting(https://michaelcurrin.github.io/coding-blog/).
```
Comment (MichaelCurrin) [inline README-it.md:21]:
```suggestion
Maggiori informazioni sulla mia esperienza e sulla mia storia sul mio blog [michaelcurrin.github.io/coding-blog](https://michaelcurrin.github.io/coding-blog/).
```

**Comment by alessiobenvenuti** [comment]
Hi, @MichaelCurrin , I added the required changes. Could you check, please? :blush:

**Pull request merged**